p2p/__init__.py

```diff
@@ -1430,43 +1431,44 @@ def _check_for_errors(self, resp, req_url):
             log.debug("[P2P][RESPONSE] %s" % request_log)
 
         if resp.status_code >= 500:
+            response_text = resp.text
```

Comment:
I'm sure there's a good reason, but why are we moving from resp.content to resp.text?